apps/admin-x-activitypub/src/views/Inbox/components/Reader.tsx

```diff
@@ -154,12 +151,12 @@ const ArticleBody: React.FC<{
                     waitForImages();
 
                     const script = document.createElement('script');
-                    script.src = '${siteUrl ? `${siteUrl.replace(/\/$/, '')}/public/cards.min.js` : '/public/cards.min.js'}';
+                    script.src = 'cards/admin-cards.min.js';
```

Comment:
This stuck out initially because it makes the script URL dependent on the currently-viewed page, _but_ I think this will be fine as long as we're using `#` URLs for the SPA

Reply:
Switched to absolute URLs in https://github.com/TryGhost/Ghost/pull/24746/commits/3d0d206df1bef0b296f3919d809ab6a74600bf39